src/metrics_server/index.ts

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+export * from './model';
```

Comment:
Can't the the other packages import directly from model?

This file is supposed to be an application, not a module. Mixing those functions is problematic. We should revert this.

Maybe we need a different structure for the packages.

Reply:
Done